manifests/04-operator.yaml

```diff
@@ -44,7 +44,7 @@ spec:
             value: "node-tuning"
           - name: RESYNC_PERIOD
             value: "600"
-          - name: CLUSTER_NODE_TUNED_IMAGE
+          - name: CLUSTER_NODE_TUNED_IMAGE_RHEL7
```

Comment:
Will the config have both keys `CLUSTER_NODE_TUNED_IMAGE_RHEL7` and `CLUSTER_NODE_TUNED_IMAGE_RHEL8`

Reply:
Eventually yes.